code/modules/projectiles/guns/energy/nuclear.dm

```diff
@@ -1,27 +1,29 @@
-/obj/item/weapon/gun/energy/gun/nuclear
+/obj/item/weapon/gun/energy/nuclear
 	name = "Prototype: advanced energy gun"
-	desc = "An energy gun with an experimental miniaturized reactor."
+	desc = "An energy handgun with an experimental miniaturized reactor. Able to fire in two round bursts."
```

Comment:
2-round?

Reply:
Should I write 2-round instead of two round? I thought it'd be better to to say two round.